test/ExtendedXmlSerializerTest/TestObject/TestClassCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// MIT License
+// 
+// Copyright (c) 2016 Wojciech Nagórski
```

Comment:
Cool. 😎 Works for me!  Although, I wouldn't quite classify it as "many". Wait until you see my _next_ PR heh heh.